packages/react-instantsearch/src/core/findResultsState.js

```diff
@@ -0,0 +1,16 @@
+/** 
```

Comment:
This is the only part I found really annoying. Couldn't succeed to add some js-doc directly under `createInstantSearchServer`. Ended up writing the findResults doc here.